target/linux/ipq40xx/image/Makefile

```diff
@@ -218,6 +218,24 @@ define Device/avm_fritzbox-4040
 endef
 TARGET_DEVICES += avm_fritzbox-4040
 
+define Device/pangubox-pgb-m5
+	$(call Device/FitImage)
+	DEVICE_VENDOR := PanguBox
+	DEVICE_MODEL := PGB-M5
+	SOC := qcom-ipq4019
+	BLOCKSIZE := 128k
+	PAGESIZE := 2048
+	DTB_SIZE := 65536
+	IMAGE_SIZE := 20439364
+	UBINIZE_OPTS := -E 5
+	FILESYSTEMS := squashfs
+	IMAGES := nand-factory.bin nand-sysupgrade.bin
+	IMAGE/nand-factory.bin := append-ubi
+	DEVICE_PACKAGES :=  -kmod-ath10k-ct kmod-ath10k-ct-smallbuffers ath10k-firmware-qca9984-ct  \
```

Comment:
Why smallbuffers?

Reply:
It was modified to ath10k-ct in new commit.